pubspec.yaml

```diff
@@ -99,6 +99,14 @@ melos:
         - format
       description: Run all static analysis checks.
 
+    analyze:
+      run: melos exec dart analyze .
+      description: Run `dart analyze` for all packages.
+
+    format:
```

Comment:
Format should still exist as a default I think

Reply:
yeah, right